gensim/models/sldamodel.py

```diff
@@ -0,0 +1,223 @@
+#!/usr/bin/env python
```

Comment:
A lot of PEP8 issues, please look [here](https://travis-ci.org/RaRe-Technologies/gensim/jobs/282716651#L1018).

Reply:
+ Missed `sldamodel.rst` in `docs/src/models`